zebra-state/src/service/finalized_state/disk_format/tests/snapshots/orchard_note_commitment_tree_raw_data@mainnet_1.snap

```diff
@@ -7,8 +7,4 @@ expression: cf_data
     k: "000000",
     v: "0000",
   ),
```

Comment:
Optional: it might reduce CPU usage to calculate and cache the root hash of the empty tree in the genesis block. Currently, every time the empty tree is accessed, the RPCs will have to calculate the root.

This can be implemented by calling `tree.root()` before writing the genesis tree to the state.

We could make this change for new trees in this PR, and by overwriting the genesis entry in the PR that upgrades existing states.

Reply:
We compute (and cache) the root of the genesis block already, but then we store `default()` trees anyway: https://github.com/zcashfoundation/zebra/blob/9ebd56092bcdfc1a09062e15a0574c94af37f389/zebra-state/src/service/finalized_state/zebra_db/shielded.rs#L365-L379. It should be an easy fix. I'll open an issue.

Reply:
This is technically a database format change, but since the deserialized trees are indistinguishable, I think we can do the fix at any time.

Reply:
Maybe with a patch format version 🙂

Reply:
I just realized I misread the last sentence in your comment. I'll do it in this PR.

Reply:
Done in [e19f365](https://github.com/ZcashFoundation/zebra/pull/7266/commits/e19f365d98f240bdfbe7a570bbf65059945e532d).